conftest.py

```diff
@@ -78,12 +79,31 @@ def appliance_police():
         port_results = {pn: net_check(pp) for pn, pp in ports.items()}
```

Comment:
I was thinking similar things, but I'm interested in getting this fixed ASAP so the test runs can finish on their own. At the moment, only port 443 has ever failed in this check, and then it's only ever failed because the UI worker has crashed and is restarting, or the UI worker has frozen up.